3ds2-example/package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "payments-auth-integration-examples",
+  "version": "1.0.0",
+  "description": "Fullstack demo app for Adyen Payments Authentication integrations.",
```

Comment:
change name to "Fullstack demo app that showcases Adyen's 3DS2 native and redirect flows."